sdk/RELEASE.md

```diff
@@ -14,11 +14,10 @@
 ## Documentation Updates
 # 1.8.17
 ## Major Features and Improvements
-
+* Allow ephemeral storage to be set via component output parameter [#8428](https://github.com/kubeflow/pipelines/pull/8428)
```

Comment:
Probably not a major feature or improvement. I would probably categorize it as other changes. But I don't have a strong opinion here.

Reply:
Agreed. Done.